stories/examples/pca.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import { Meta } from '@storybook/react';
+
+import { Axis, Plot, Heading, ScatterSeries, Annotations } from '../../src';
+import { Ellipse } from '../../src/components/Annotations/Ellipse';
+import { Group } from '../../src/components/Annotations/Group';
+import { Text } from '../../src/components/Annotations/Text';
+import data from '../data/pca.json';
+import { DEFAULT_PLOT_CONFIG } from '../utils';
+
+export default {
+  title: 'Examples/PCA example',
+} as Meta;
+interface Point {
+  x: number;
+  y: number;
+  color: {
+    shape: string;
+    fill: string;
+    r: number;
+  };
+  id: string;
+}
+export function PCAExample() {
+  return (
+    <Plot {...DEFAULT_PLOT_CONFIG}>
+      <Heading title="COVID-19 cases in USA (2020)" />
+      <ScatterSeries
+        markerStyle={{
+          fill: (point: Point) => point.color.fill,
+          stroke: 'none',
+        }}
+        data={data.points}
+        xAxis="x"
+        yAxis="y"
+      />
+
+      <Annotations>
+        {data.ellipses.map(
+          ({ x, y, rx, ry, fillColor, fillOpacity, label, angle }) => (
+            <Group
+              key={`${x}-${y}`}
+              x={x}
+              y={y}
+              horizontalAlign="middle"
+              verticalAlign="middle"
+            >
```

Comment:
Does it work without group ? I would just create ellipse and text elements with the corresponding `x`, `y` values. I'm afraid that if we have a big text by using horizontal / vertical align we will move the center of the ellipse and this should never happen.

Reply:
I would also implement it without a group

Reply:
it doesn't work without group
![image](https://user-images.githubusercontent.com/56769876/149186655-de5f6734-0298-4fc0-a97e-731717761f24.png)

Reply:
the problem was in the rotation

Reply:
I see. Maybe we should add a 4th option to the align props of the Annotation groups: `none`, which would not render a GroupAlign, but just a `g` ?

Reply:
In this case , when i don't use group the transfer origin is the same as the plot (this is why the rotation is not done correctly)

Reply:
Well this is the reason this example was created. We knew that there could be problem with rotation.

I think when we rotate we always want to do it using the center of the shape or group as origin. I wonder if this could help: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/transform-origin (transform-origin: center)

Reply:
Yes i already used transform origin in te style but we still have the same problem
![image](https://user-images.githubusercontent.com/56769876/149191215-4cdb7fe8-9c26-4600-b7ce-5c425da0c647.png)

Reply:
Seems adding ` transform-box: fill-box;  ` could help

I did this testcase

```svg
<svg height="500" width="500">
  <ellipse cx="100" cy="100" rx="100" ry="20" style="fill:yellow" />
  <ellipse cx="200" cy="200" rx="100" ry="20"
   style="fill:white; transform-box: fill-box; transform-origin: center; transform:rotate(45deg);"/>
  <ellipse cx="200" cy="200" rx="100" ry="20" style="fill:white; "/>
</svg>
```

![image](https://user-images.githubusercontent.com/1484241/149193100-3ed975c9-6864-4e1c-9f79-f57bb1f2cc05.png)

Reply:
I don't think we need such tricks. It works fine with a simple `g` (the point of reference becomes the position of the group):

![image](https://user-images.githubusercontent.com/2352663/149194032-e4b24284-672b-4a5d-aa7a-69e0c274057d.png)

Reply:
I only changed this to test my hypothesis:
![image](https://user-images.githubusercontent.com/2352663/149194189-6c0c76b2-13c3-4a42-820a-0f5458783cc0.png)

Reply:
yes it's working thank you

Reply:
> I only changed this to test my hypothesis: ![image](https://user-images.githubusercontent.com/2352663/149194189-6c0c76b2-13c3-4a42-820a-0f5458783cc0.png)

Do you want me to use it?

Reply:
> I see. Maybe we should add a 4th option to the align props of the Annotation groups: `none`, which would not render a GroupAlign, but just a `g` (it could be the default behavior) ?

So we can add this option to Group annotation

Reply:
> > I see. Maybe we should add a 4th option to the align props of the Annotation groups: `none`, which would not render a GroupAlign, but just a `g` (it could be the default behavior) ?
> 
> So we can add this option to Group annotation

I created https://github.com/zakodium/react-d3-utils/issues/16 for this.